postgres/schema.sql

```diff
@@ -1,6 +1,6 @@
 CREATE TABLE diagnosis_keys
 (
-    key uuid NOT NULL,
+    key bytea NOT NULL,
```

Comment:
For now, existing databases need to be updated manually. Existing data _should_ persist, but haven't tried.
For local env, wipe your Docker volume (e.g. `docker-compose down --volumes` or manually).